core/commands/storage.go

```diff
@@ -583,29 +605,42 @@ func payFullToEscrow(response *escrowpb.SignedSubmitContractResult, configuratio
 		log.Error(err)
 		return
 	}
-	err = escrow.PayInToEscrow(configuration, payinRequest)
+	payinRes, err := escrow.PayInToEscrow(configuration, payinRequest)
 	if err != nil {
 		log.Error(err)
 		return
 	}
+	// get escrow sig, add them to guard
+	sig := payinRes.EscrowSignature
+	for _, guardContract := range guardContracts {
+		guardContract.EscrowSignature = sig
+		guardContract.EscrowSignedTime = time.Now()
```

Comment:
why set it on go-btfs node?? shouldn't this come from escrow?

Reply:
I don't think escrow return time to node, I can only put escrow's sig on guard contract and using the write time as signed time

Reply:
find it! will add